scalameta/trees/shared/src/main/scala/scala/meta/Trees.scala

```diff
@@ -438,6 +439,8 @@ object Ctor {
     self: Self,
     stats: List[Stat]
 ) extends Tree {
+  @binaryCompatField
+  private var derives: List[Init] = Nil
```

Comment:
derives can only be `QualId` which is `a.b.c.Element` and Init has also name / argss. Maybe something more narrow like `Type`? Only first field of `Init`.

Reply:
Changed!